### ISSUE TITLE
Do you plan to release a pre-trained model?

### ISSUE THREAD

**Issue by SugarSpinda** [hackiey/keypoints#2]
Hi @hackiey 

Do you plan to release a pre-trained model,
or training/validation loss, or average precision?

Thanks.

**Comment by hackiey** [comment]
Hi, @CltKitakami I'm very sorry for some confusion caused by this repository, actually I use this paper's method to do another project, so I didn't test the performance on the paper's dataset. I just analyze an image with visualization.
